src/components/Person.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+function Person(props) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = props;
+
+  return (
+    <section className="Person">
+      <h1 className="Person__name">
+        {`My name is ${name}`}
+      </h1>
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried
+          ? `My ${sex === 'm' ? `wife's` : `husband's`} name is ${partnerName}`
+          : 'I am not married'}
+      </p>
```

Comment:
It's better to create a separate variable before return statement where you can store the result of inner conditional statement (checking sex === 'm')
```suggestion
      <p className="Person__partner">
        {isMarried
          ? `My ${partner}'s name is ${partnerName}`
          : 'I am not married'}
      </p>
```